aspnetcore/fundamentals/middleware/sample/Chain/project.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.1",
+      "type": "platform"
+    },
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
```

Comment:
When I change to 1.1.0 I get the following  502.5 :

Common causes of this issue: 
The application process failed to start 
The application process started but then stopped 
The application process started but failed to listen on the configured port